nautobot_chatops_ipfabric/worker.py

```diff
@@ -452,7 +469,9 @@ def pathlookup(
     snapshot_id = get_user_snapshot(dispatcher)
     sub_cmd = "pathlookup"
     supported_protocols = ["tcp", "udp", "icmp"]
+    # supported_flags = ["ack", "fin", "psh", "rst", "syn", "urg"]
```

Comment:
why do we add this and tcp_flags and comment them out?

Reply:
I think because ipfabric-diagrams does validation check on the flags.

Reply:
will remove, tested something to add them in but ran into a bug and will need another approach.